app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] As I already told you that the author is the alias of the user then it would be better if you make the belongs_to relationship of the post with the author 👍 After making the changes it would look like the below ⬇️

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```